scripts/instr_trace_compare.py

```diff
@@ -131,8 +135,8 @@ def compare_trace_csv(csv1, csv2, name1, name2, log,
           for trace_1_index in range(0, len(gpr_trace_1[gpr])-1):
             if (trace_2_index == len(gpr_trace_2[gpr])):
               break
-            if long(gpr_trace_1[gpr][trace_1_index].rd_val, 16) != \
-               long(gpr_trace_2[gpr][trace_2_index].rd_val, 16):
+            if int(gpr_trace_1[gpr][trace_1_index].rd_val, 16) != \
```

Comment:
Is "long" causing problem here? I am not sure if your change will work for XLEN = 64 case

Reply:
yes, using "long" is causing python3 to error out. It looks like the "long" and "int" types were unified in Python 3.x